samples/sampler/stories/qa/Dropdown.js

```diff
@@ -1,88 +1,176 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
-import {number, text} from  '@enact/storybook-utils/addons/knobs';
+import {boolean, select, text} from  '@enact/storybook-utils/addons/knobs';
 import {action} from '@enact/storybook-utils/addons/actions';
 import {storiesOf} from '@storybook/react';
 import React from 'react';
 
+import Button from '@enact/agate/Button';
 import Dropdown, {DropdownBase} from '@enact/agate/Dropdown';
-import Scroller from '@enact/agate/Scroller';
-import Group from '@enact/ui/Group';
-import ri from '@enact/ui/resolution';
+import Heading from '@enact/agate/Heading';
 
 const Config = mergeComponentMetadata('Dropdown', Dropdown, DropdownBase);
+const items = (itemCount, optionText = 'Option') => (new Array(itemCount)).fill().map((i, index) => `${optionText} ${index + 1}`);
 Dropdown.displayName = 'Dropdown';
 
-storiesOf('Dropdown', module)
-	.add(
-		'with 4 directions',
-		() => {
-			const itemCount = number('items', Config, {range: true, min: 0, max: 50}, 5);
-			const items = (new Array(itemCount)).fill().map((i, index) => `Option ${index + 1}`);
+const list = [
+	{children: 'hello 1', 'key': 'key1', 'aria-label': 'aria 1'},
+	{children: 'hello 2', 'key': 'key2', 'aria-label': 'aria 2', disabled: true},
+	{children: 'hello 3', 'key': 'key3', 'aria-label': 'aria 3'}
+];
 
-			return (
-				<>
-					<Dropdown
-						direction="left"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: 0, right: 0, width:'25%'}}
-						title="Left dropdown"
-					>
-						{items}
-					</Dropdown>
-					<Dropdown
-						direction="right"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: 0, left: 0, width:'25%'}}
-						title="Right dropdown"
-					>
-						{items}
-					</Dropdown>
-					<Dropdown
-						direction="down"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: 0, left: '50%', transform: 'translateX(-50%)', width:'25%'}}
-						title="Down dropdown"
-					>
-						{items}
-					</Dropdown>
-					<Dropdown
-						direction="up"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: ri.scaleToRem(440),  width:'25%'}}
-						title="Up dropdown"
-					>
-						{items}
-					</Dropdown>
-				</>
-			);
-		},
-		{
-			text: 'The basic Dropdown'
-		}
-	).add(
-		'Group in Scroller',
```

Comment:
This one should not be removed. I think it has a test case.

Reply:
ah, right 👍